CHANGELOG.md

```diff
@@ -7,13 +7,17 @@ accidentally triggering the load of a previous DB version.**
 ## 1.7.0 (unreleased)
 
 **Major Features**
-* #1685 Add real-time support to continuous aggregates
 * #1456 Add support for PostgreSQL 12
+* #1685 Add support for real-time aggregates
 
 **Minor Features**
 
 **Bugfixes**
 * #1665 Add ignore_invalidation_older_than to timescaledb_information.continuous_aggregates view
+* #1750 Handle undefined ignore_invalidation_older_than
+* #1757 Restrict watermark to max for continuous aggregates
+* #1769 Add rescan function to CompressChunkDml CustomScan node
+* #1785 Fix last_run_success value in continuous_aggregate_stats view
```

Comment:
I would add a mention of this contribution: https://github.com/timescale/timescaledb/pull/1808 and maybe a note under "Thanks"

Reply:
done